### PR TITLE
refactored constants, moved predefined MSDN constns to another file.

### PR DESCRIPTION
**Pull request by Nimesh-Msys**
Signed-off-by: Nimesh Patni <nimesh.patni@msystechnologies.com>

### CODE REVIEW

**Comment by djberg96** [comment]
:+1:

**Comment by btm** [comment]
@Nimesh-Msys please rebase.

**Comment by Nimesh-Msys** [comment]
@btm Done. Could you please take a look

File: lib/win32/taskscheduler.rb [hunk 1]
```diff
@@ -10,250 +11,106 @@ module Win32
   # The TaskScheduler class encapsulates a Windows scheduled task
   class TaskScheduler
     include Windows::TaskSchedulerHelper
-
-    # The version of the win32-taskscheduler library
-    VERSION = '0.3.2'.freeze
```
Comment (btm) [inline lib/win32/taskscheduler.rb]: @Nimesh-Msys I'm going to merge this, but consider moving this constant back. This is specific to the library and let's you ask `Win32::TaskScheduler::VERSION`. I think moving it is an unnecessary breaking change.

Comment (btm) [reply lib/win32/taskscheduler.rb]: @Nimesh-Msys we're going to want to bump this to 1.0.0 with all the changes too.

**Pull request merged**